LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Lauren Elizabeth Tan
+Copyright (c) 2018 Lauren Elizabeth Tan
```

Comment:
This is a classical `ember-cli-update` issue with unintentional update of a copyright year :laughing: 
I think it should be reverted